### DISCUSSION TITLE
In Vim mode, goto directory in project panel

### DISCUSSION THREAD

**Discussion by akhildevelops** [zed-industries/zed#20370] [Feature Requests]
Is there a way to go to deeply nested `foo` directory while cursor is present in project panel at root level.

```shell
project
|--cargo.toml
|--foo
    |--apple.json
    |--baz
        |--orange.json
        |--foo
            |--pineapple.json
        

```

Looking something like below:
```shell
ctrl+shift+e #Goto project panel.
/foo+<enter> #Goto foo.
n #Goto next directory with same name foo.
```

**Comment by 0x2CA** [comment]
Hey, I think you want a filter to filter the search project panel
As far as I know, it's not available now

The outline panel has similar functionality, but doesn't have a good keymap for vim

https://github.com/user-attachments/assets/a64f4f20-6a73-4b0f-936b-370bc40a82f3

**Reply by akhildevelops** [reply]
Does it filters directories as I can see symbols from the video.

Similar to nerd tree, if zed can navigate project panel like below would be cool.

Tree navigation
p: go to parent
P: go to root
K: go to first child
J: go to last child
<C-­k>: go to prev sibling
<C-­j>: go to next sibling

https://cheatography.com/stepk/cheat-sheets/vim-nerdtree/

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
